indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -438,20 +454,70 @@ function EditableListDisplay({
       />
     );
   };
+  const renderTags = (__, editable) => {
+    const tags = editable.tags.sort((a, b) => a.code.localeCompare(b.code));
+    const labelTags = () => {
+      return (
+        <div style={{display: 'flex', flexWrap: 'nowrap'}}>
+          {tags.map(t => {
+            return (
+              <Label key={t.code} size="small" color={t.color}>
+                {t.code}
+              </Label>
+            );
+          })}
+        </div>
+      );
+    };
+
+    const colorTags = () => {
+      return (
+        <div>
+          {tags.map(t => {
+            return <Label key={`${t.code}-dot`} circular empty color={t.color} />;
+          })}
+        </div>
+      );
+    };
+
+    return [labelTags(), colorTags()];
+  };
   const renderFuncs = {
     friendlyId: renderId,
     title: renderTitle,
     code: renderCode,
     revision: editable => (editable ? editable.revisionCount : ''),
     status: renderStatus,
     editor: renderEditor,
+    tags: renderTags,
   };
 
   // eslint-disable-next-line react/prop-types
   const renderCell = ({dataKey, rowIndex}) => {
     const row = sortedList[rowIndex];
     const fn = renderFuncs[dataKey];
-    return (
+
+    return dataKey === 'tags' && row.editable?.tags ? (
```

Comment:
You need to handle the case when `dataKey === 'tags'` is true, but either the tags or the editable does not exist. Currently it goes to the else branch of the ternary where it crashes on line 458 in `renderTags`